google/gmail/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM ghcr.io/astral-sh/uv:python3.13-bookworm-slim
+
+# Prevents Python from writing .pyc files and buffers stdout/stderr
+ENV PYTHONDONTWRITEBYTECODE=1 \
+    PYTHONUNBUFFERED=1
+
+WORKDIR /app
+
+# Install system deps (if any needed) - using apt for Debian
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    build-essential \
+    libpq-dev \
+    ffmpeg \
+    && rm -rf /var/lib/apt/lists/*
+
+# Copy uv project files first for better Docker layer caching
+COPY pyproject.toml uv.lock README.md ./
+
+# Install dependencies using uv sync (faster and more reliable)
+RUN uv sync --frozen --no-dev
+
+# Copy your application code
+COPY . .
+
+# Expose the port using the PORT environment variable
+ENV PORT=9000
+EXPOSE $PORT
```

Comment:
Is this valid? I'm not sure this will actually work. Both because I don't think `EXPOSE` will accept the substitution, but also because it is static at build time, where `ENV` is for runtime stuff.